src/main/Main.ts

```diff
@@ -1,12 +1,50 @@
-import {app, ipcMain, nativeImage, BrowserWindow, screen} from "electron";
+import {app, nativeImage, BrowserWindow, screen, globalShortcut, Menu} from "electron";
 import {readFileSync} from "fs";
 import {windowBoundsFilePath} from "../utils/Common";
+import {buildMenuTemplate} from "./Menu";
+
+/**
+ * Contains references to all currently open windows.
+ *
+ * @type {Array}
+ */
+let windows: Electron.BrowserWindow[] = [];
+
+/**
+ * The index of the currently focussed window in `windows`.
+ *
+ * @type {number}
```

Comment:
@jsphpl typescript is aware of the params so jsdoc is no longer needed... You can just explain what the parameter does. But you don't need to specify any types on the documentations...